fe/fe-core/src/main/java/com/starrocks/analysis/DescriptorTable.java

```diff
@@ -44,7 +44,6 @@
  * them unique ids..
  */
 public class DescriptorTable {
-    private static final Logger LOG = LogManager.getLogger(DescriptorTable.class);
```

Comment:
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

should be removed too

Reply:
get，thanks